src/Input.js

```diff
@@ -5,7 +5,7 @@ import deprecationWarning from './utils/deprecationWarning';
 
 class Input extends InputBase {
   render() {
-    if (this.props.type === 'static') { // eslint-disable-line react/prop-types
+    if (this.props.type === 'static') {
```

Comment:
o great :). there is a lot of benefit to the double branches, we landed a bunch of breaking changes while still being able to make patch releases of the current branch, otherwise we'd have to finish all new work just to release bug fixes!